### PR TITLE
feat(init-1D): #MVS-127 add timeline notification for the end process

### PR DESCRIPTION
**Pull request by flmartineau**
## Describe your changes

- added success/error timeline notification at the end of the initialization process
- fixed jsonSchema

## Checklist tests

- check that a new notification is added after the initilization is complete

## Issue ticket number and link

https://jira.support-ent.fr/browse/MVS-127

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/viescolaire/helper/UserInfosHelper.java
Comment (flmartineau) [inline src/main/java/fr/openent/viescolaire/helper/UserInfosHelper.java:1]: Fichier récupéré depuis Présences pour transformer un JSON en UserInfos :
https://github.com/OPEN-ENT-NG/presences/blob/dev/common/src/main/java/fr/openent/presences/common/helper/UserInfosHelper.java

File: src/main/java/fr/openent/viescolaire/worker/InitWorker.java [hunk 1]
```diff
@@ -69,7 +77,33 @@ protected void run(Message<JsonObject> event) {
...
+        Promise<Void> promise = Promise.promise();
+
+        JsonObject params = new JsonObject()
+                .put(Field.PUSHNOTIF, new JsonObject()
+                    .put(Field.TITLE,"push.notif.viescolaire.new.notification")
+                    .put(Field.BODY, ""));
+
+        timelineHelper.notifyTimeline(null, isError ? "viescolaire.initialization_error"
+                        : "viescolaire.initialization_done", user,
+                Collections.singletonList(user.getUserId()), user.getUserId(), params);
```
Comment (Sofianebernoussi) [inline src/main/java/fr/openent/viescolaire/worker/InitWorker.java:104]: Pourquoi avoir besoin de tout ce helper de user etc... juste pour un id user? et c'est normal que l'user soit et l'expediteur et le mec qui recoit la notif?

Comment (flmartineau) [reply src/main/java/fr/openent/viescolaire/worker/InitWorker.java:104]: notifyTimeline prend un userInfos en paramètre. C'est une fonction de l'entcore.

Oui c'est l'utilisateur qui lance l'intitialisation qui reçoit la notification.

Comment (Sofianebernoussi) [reply src/main/java/fr/openent/viescolaire/worker/InitWorker.java:104]: oui mais c'est pour avoir le sender, dans la plupart des modules on le met a null je pense, et l'id de l'user tu l'as aisément

**Review by Sofianebernoussi — APPROVED** [review]

**Review by BastienBenard — APPROVED** [review]

**Pull request merged**